apps/shipments/signals.py

```diff
@@ -22,10 +32,11 @@ def shipment_job_update(sender, message, listener, **kwargs):
 
 @receiver(event_update, sender=Shipment, dispatch_uid='shipment_event_update')
 def shipment_event_update(sender, event, listener, **kwargs):
-
-    # TODO: Metrics for processed Events
+    LOG.debug(f'Shipment event update with sender {sender}.')
```

Comment:
The `sender` of this signal is just a `Shipment` class so this log isn't really informative. Maybe log `event.id` (Event.id) and/or `listener.id` (Shipment.id) instead